docs/index.md

```diff
@@ -4,9 +4,244 @@ hero:
   name: The design system for Schibsted Marketplaces
   text:
   tagline: Design, build and ship coherent experience with WARP
-  image: /warp-logo.svg
+  image: /warp-hero.svg
   actions:
   - theme: brand
     text: Get Started →
     link: /get-started/
+
+features:
+  - icon: 
+      src: /components-icon.svg
+      width: 158px
+      height: 162.17px
+      alt: UI mockup with multiple horizontal bars representing text, buttons and icons, alongside small colored squares and circular icons at the bottom with one indicating a checkmark and another showing a double-arrow icon.
+    title: Components
+    details: UI controls and utilities to help you build great experiences
+    link: /components
+    linkText: See all components 
+
+  - icon: 
+      src: /foundations-icon.svg
+      width: 126px
+      height: 181px
+      alt: Abstract illustration with red circles, geometric shapes, and stylized figures.
+    title: Foundations
+    details: Guidelines for colour palettes, typography, icons and more.
+    link: /foundations
+    linkText: Explore foundations
 ---
+
+<script setup>
+const baseUrl = import.meta.env.BASE_URL
+
+const componentData = [
+  {
+    title: 'CSS classes',
+    href: `${baseUrl}foundations/styling/web/unocss`,
```

Comment:
It should work if you remove `baseUrl` and keep the link as `foundations/styling/web/unocss` (so without a `/` at the beginning. That will append the path to the existing url, which at this point is `http://.../warp-portal-poc/`. The same applies to the rest of the href values in this file.

Reply:
Updated in this commit: [cb864f94](https://github.com/warp-ds/warp-portal-poc/pull/10/commits/cb864f94ee1ba26ea583e9e24e41ee74a60c9461)